File: solutions/typescript/darts/1/darts.ts
export function score(x: number, y: number): number {
  const euclidean_distance = Math.hypot(x, y)

  if (euclidean_distance <= 1) return 10
  else if (euclidean_distance <= 5) return 5
  else if (euclidean_distance <= 10) return 1
  return 0
}

File: solutions/python/darts/1/darts.py
import math


def score(x: int, y: int) -> int:
    euclidean_distance = math.hypot(x, y)
    if euclidean_distance <= 1:
        return 10
    if euclidean_distance <= 5:
        return 5
    if euclidean_distance <= 10:
        return 1
    return 0
